laws/functor.js

```diff
@@ -12,18 +11,18 @@ const {map} = require('..');
 
 */
 
-const identityʹ = t => eq => x => {
-    const a = t(x)[map](identity);
-    const b = t(x);
+const identity = of => eq => x => {
+    const a = of(x)[map](x => x);
+    const b = of(x);
     return eq(a, b);
 };
 
-const composition = t => eq => x => {
-    const a = t(x)[map](compose(identity)(identity));
-    const b = t(x)[map](identity)[map](identity);
+const composition = of => eq => x => f => g => {
+    const a = of(x)[map](x => f(g(x)));
```

Comment:
This more closely matches what appears in the spec. Also, the `fantasy-combinators` dependency seems unnecessary to me, since `x => f(g(x))` and the other expressions it allows us to write without points are not verbose to begin with.